scripts/utility.py

```diff
@@ -445,7 +445,11 @@ def event_text_adjust(Cat,
     if "acc_singular" in adjust_text:
         adjust_text = adjust_text.replace("acc_singular", str(plural_acc_names(cat.accessory, False, True)))
 
-    adjust_text = adjust_text.replace("c_n", str(clan.name) + "Clan")
```

Comment:
I suggest removing game.clan as the default value for the clan parameter. Python default parameter are set by-value at function declaration, so it will always be none.